server/controllers/api.py

```diff
@@ -885,6 +889,17 @@ def post(self, user, backup_id):
 
         return self.schema.store_comment(user, backup)
 
+class File(Resource):
+    """ Redirect (or download) a file. No Schema due to redirect
+        Authenticated. Permissions: >= User/Staff
+        Used by: Course Scripts.
+    """
+    schema = None
+    model = models.ExternalFile
+
+    def get(self, user, file_id):
+        return files.file_download(file_id, user)
```

Comment:
In a truly RESTful API this would probably have to return a JSON with another URL to download the file - but I don't mind ignoring REST so that we can use the simple solution.